pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -108,6 +108,8 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.Address = kubeAddress
 	server.Port = uint(kubePort)
 	server.ReadOnlyPort = 0 // no read only access
+	server.CadvisorPort = 0 // no unsecured cadvisor access
+	server.HealthzPort = 0  // no unsecured healthz access
```

Comment:
healthz is already surfaced on 10250. Didn't know if we needed it standalone and unsecured as well